packages/commonwealth/client/scripts/views/components/feed.tsx

```diff
@@ -34,13 +34,13 @@ export const Feed = ({
   const [data, setData] = useState<DashboardActivityNotification[]>();
   const [labels, setLabels] = useState<(IEventLabel | undefined)[]>();
   const [currentCount, setCurrentCount] = useState<number>(
-    defaultCount || DEFAULT_COUNT
+    defaultCount || DEFAULT_COUNT,
```

Comment:
how about this? did you fix your prettier? there should not be comma at the EOL here